WpfMath/TexFormulaParser.cs

```diff
@@ -65,6 +67,8 @@ internal static string[][] DelimiterNames
 
         public static void Initialize()
         {
+            if (isInitialized) return;

```

Comment:
We don't need that `isInitialized` variable at all; we're initializing everything in a static constructor that's
1) guaranteed to be executed
2) guaranteed to be executed in a thread-safe way

Please remove that variable.